spec/mail/network/delivery_methods/stdout_spec.rb

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+require 'spec_helper'
+
+describe "Stdout Delivery Method" do
+
+  before(:each) do
+    # Reset all defaults back to original state
+    Mail.defaults do
+      delivery_method :smtp, { :address              => "localhost",
```

Comment:
If you 2-space indent everything there is less space wasted and nothing needs to change when the first line changes:
```Ruby
delivery_method :smtp, 
  :address              => "localhost",
  :port                 => 25,
  :domain               => 'localhost.localdomain',
  :user_name            => nil,
  :password             => nil,
  :authentication       => nil,
  :enable_starttls_auto => true
```